osbs/cli/main.py

```diff
@@ -352,7 +352,7 @@ def cmd_build(args, osbs):
         required_args = ['platform', 'release']
         unwanted_args = ['platforms']
         create_func = osbs.create_worker_build
-    elif args.orchestrator:
+    elif args.orchestrator and osbs.can_orchestrate():
```

Comment:
>can_orchestrate is option which allows it at all,
--orchestrator is just saying I want to do it

Ah, yes, agree. Currently k-cb assumes that if OSBS can orchestrate it will orchestrate - is this still true? Would any k-cb changes be required after this PR is merged?